lean4-infoview/src/infoview/index.css

```diff
@@ -135,3 +143,168 @@ select {
     color: var(--vscode-editorWidget-foreground);
     background: var(--vscode-editorWidget-background);
 }
+
+.hover-div {
+    overflow: hidden; font-size: 14px; line-height: 1.35714;
+}
+
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+ /* from github.com/microsoft/vscode/src/vs/base/browser/ui/hover/hover.css */
+ .monaco-hover {
+	cursor: default;
+	overflow: hidden;
+	z-index: 50;
+	user-select: text;
+	-webkit-user-select: text;
+	-ms-user-select: text;
+	box-sizing: initial;
+	line-height: 1.5em;
+}
+
+.monaco-hover.hidden {
+	display: none;
+}
+
+.monaco-hover a:hover {
+	cursor: pointer;
+}
+
+.monaco-hover .hover-contents:not(.html-hover-contents) {
+	padding: 4px 8px;
+}
+
+.monaco-hover .markdown-hover > .hover-contents:not(.code-hover-contents) {
+	/* max-width: 500px; */
+	word-wrap: break-word;
+}
+
+.monaco-hover .markdown-hover > .hover-contents:not(.code-hover-contents) hr {
+	/* min-width: 100%; */
+}
+
+.monaco-hover p,
+.monaco-hover .code,
+.monaco-hover ul {
+	margin: 8px 0;
+}
+
+.monaco-hover code {
+	font-family: var(--monaco-monospace-font);
+}
+
+.monaco-hover hr {
+	box-sizing: border-box;
+	border-left: -8px;
+	border-right: -8px;
+	margin-top: 4px;
+	margin-bottom: 4px;
+	margin-left: -18px;
+	margin-right: -18px;
+	height: 1px;
+    border-bottom-width: 0px;
+    border-top-width: 1px;
+	border-top-style: solid;
+	border-top-color: var(--vscode-editorWidget-border);
+}
+
+.monaco-hover p:first-child,
+.monaco-hover .code:first-child,
+.monaco-hover ul:first-child {
+	margin-top: 0;
+}
+
+.monaco-hover p:last-child,
+.monaco-hover .code:last-child,
+.monaco-hover ul:last-child {
+	margin-bottom: 0;
+}
+
+/* MarkupContent Layout */
+.monaco-hover ul {
+	padding-left: 20px;
+}
+.monaco-hover ol {
+	padding-left: 20px;
+}
+
+.monaco-hover li > p {
+	margin-bottom: 0;
+}
+
+.monaco-hover li > ul {
+	margin-top: 0;
+}
+
+.monaco-hover code {
+	border-radius: 3px;
+	padding: 0 0.4em;
+}
+
+.monaco-hover .monaco-tokenized-source {
+	white-space: pre-wrap;
+}
+
+.monaco-hover .hover-row.status-bar {
```

Comment:
It looks like we are importing Monaco styles wholesale :D @lovettchris , do we really need all this stuff? There are no `status-bar` or `actions` or `action-container`.

Reply:
yes I copied it all just in case we need to use those things for new features in the future, it is not very big, only about 3kb of text.